Source/Core/Core/IOS/USB/USB_HID/HIDv4.h

```diff
@@ -45,6 +45,7 @@ class USB_HIDv4 final : public USBHost
   static constexpr u8 HID_CLASS = 0x03;
 
   bool m_has_pending_changes = true;
+  bool m_is_shutdown = false;
```

Comment:
This should be `m_is_shut_down`, right? I think `shutdown` as one word only exists as the action itself.

Reply:
Yeah it should be, will add this in now